.docker/entrypoint.sh

```diff
@@ -1,10 +1,10 @@
 #! /bin/bash
 
 # print folder contents for debugging
-echo "Contents:"
-echo ""
+printf "Contents:"
+printf ""
```

Comment:
Just FYI, `printf` doesn't print a trailing newline by default, so this line will do nothing. If you intend to switch `echo` to `printf` throughout, you'll have to add the `\n` at the end of each output for it to match `echo`'s output.